docs/resources/dcs_redis_instance.md

```diff
@@ -0,0 +1,261 @@
+
+  backup_policy {
+    backup_type     = "auto"
+    save_days       = 5
+    period_type     = "weekly"
+    backup_at       = [1, 2, 3, 4, 5, 6, 7]
+    begin_at        = "02:00-04:00"
+  }
+
+  whitelists {
+    group_name = "group_1"
+    ip_address = ["192.168.1.0/24"]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the DCS redis instance resource.
+  If omitted, the provider-level region will be used. Changing this creates a new DCS instance resource.
+
+* `name` - (Required, String) Specifies the name of an instance.
+  The name must be 4 to 64 characters and start with a letter.
+  Only chinese, letters (case-insensitive), digits, underscores (_) ,and hyphens (-) are allowed.
+
+* `engine_version` - (Required, String, ForceNew) Specifies the version of a cache engine.
+  Currently, the valid values are `3.0`, `4.0` and `5.0`. Changing this creates a new instance.
+
+* `capacity` - (Required, Float) The total memory of the cache, in GB.
+  + **Redis4.0 and Redis5.0**: Stand-alone and active/standby type instance values: `0.125`, `0.25`, `0.5`, `1`, `2`,
+    `4`, `8`, `16`, `32` and `64`.
+    Cluster instance specifications support `24`, `32`, `48`, `64`, `96`, `128`, `192`, `256`, `384`, `512`, `768` and
+    `1024`.
+  + **Redis3.0**: Stand-alone and active/standby type instance values: `2`, `4`, `8`, `16`, `32` and `64`.
+    Proxy cluster instance specifications support `64`, `128`, `256`, `512`, and `1024`.
+  + **Memcached**: Stand-alone and active/standby type instance values: `2`, `4`, `8`, `16`, `32` and `64`.
+
+* `resource_spec_code` - (Required, String) The specification code of the cache instance, which including the total
+  memory, available memory, maximum number of connections allowed, maximum/assured bandwidth and reference performance.
+  You can specify specification codes to create different type of Redis instances.
+  It can be obtained through this data source `huaweicloud_dcs_flavors`.
+
+* `available_zones` - (Required, List, ForceNew) The code of the AZ where the cache node resides.
+  Master/Standby, Proxy Cluster, and Redis Cluster DCS instances support cross-AZ deployment.
+  You can specify an AZ for the standby node. When specifying AZs for nodes, use commas (,) to separate AZs.
+  Changing this creates a new instance.
+
+* `vpc_id` - (Required, String, ForceNew) The ID of VPC which the instance belongs to.
+  Changing this creates a new instance resource.
+
+* `subnet_id` - (Required, String, ForceNew) The ID of subnet which the instance belongs to.
+  Changing this creates a new instance resource.
+
+* `security_group_id` - (Optional, String) The ID of the security group which the instance belongs to.
+  This parameter is mandatory for Redis 3.0 version.
+
+* `ip` - (Optional, String, ForceNew) The IP address of the DCS instance, which can only be the currently available IP
```

Comment:
How about use `pravite_ip` or `fixed_ip` which is more clear?
And the resource may support public ip in the future, use ip here could be confusing.

Reply:
it follows the old resource, it is named ip in the resource.